src/Routes.js

```diff
@@ -93,8 +93,8 @@ const Routes = (props) => (
     <AuthenticatedRoute path="/dataset_registration" component={DatasetRegistration} props={props} rolesAllowed={[USER_ROLES.admin, USER_ROLES.chairperson]} />
     <AuthenticatedRoute path="/admin_manage_lc/" component={AdminManageLC} props={props} rolesAllowed={[USER_ROLES.admin]} />
     <AuthenticatedRoute path="/admin_manage_dar_collections/" component={AdminManageDarCollections} props={props} rolesAllowed={[USER_ROLES.admin]} />
+    {(props.env === 'local' || props.env === 'dev') && <AuthenticatedRoute path="/dataset_catalog/:variant" component={CustomDatasetCatalog} props={props} rolesAllowed={[USER_ROLES.researcher]}/>}
```

Comment:
Any reason we're restricting this to the researcher role? I can envision this being important for any role, TBH. In practice, since ALL users are created with the researcher role, there's no functional difference if you don't specify other roles.

Reply:
oh, nope.  I just copy pasta'd :)